### PR TITLE
modify get ad result to return correct anomaly/feature data

### PR DESCRIPTION
**Pull request by ylwu-amzn**
*Issue #, if available:*

*Description of changes:*

* modify get ad result to return correct anomaly/feature data

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: server/models/types.ts [hunk 1]
```diff
@@ -88,26 +88,50 @@ export type GetDetectorsQueryParams = {
...
 };
 
 export type AnomalyResult = {
   startTime: number;
   endTime: number;
-  grade: number;
+  plotTime: number;
+  anomalyGrade: number;
   confidence: number;
 };
```
Comment (ohltyler) [inline server/models/types.ts:114]: do these changes break code anywhere else that uses these types?

Comment (ylwu-amzn) [reply server/models/types.ts:114]: No, this only used by `getAnomalyResults`

File: server/routes/utils/adHelpers.ts [hunk 2]
```diff
@@ -129,25 +129,21 @@ export const anomalyResultMapper = (anomalyResults: any[]): AnomalyResults => {
...
         rest.anomalyGrade != null && rest.anomalyGrade > 0
-          ? Number.parseFloat(rest.confidence).toFixed(3)
+          ? Number.parseFloat(rest.confidence).toFixed(2)
           : 0,
       startTime: rest.dataStartTime,
       endTime: rest.dataEndTime,
-      plotTime:
-        rest.dataStartTime +
-        Math.floor((rest.dataEndTime - rest.dataStartTime) / 2),
+      plotTime: rest.dataEndTime,
```
Comment (yizheliu-amazon) [inline server/routes/utils/adHelpers.ts:140]: just want to confirm, should we use data end time or data start time as plot time? Dashboard Live chart uses data start time. I guess we may keep consistency.

Comment (ylwu-amzn) [reply server/routes/utils/adHelpers.ts:140]: How about we use data end time? If we use start time, hard to compare alerts history and anomalies on AD result chart.

**Pull request merged**